lib/puppet_x/puppetlabs/cd4pe_client.rb

```diff
@@ -262,16 +262,29 @@ def web_ui_endpoint
       endpoint
     end
 
+    def connection_read_timeout
+      # Set timeout to 10min or customer override for our code deployment
+      # api as it is a long lived connection instead of polling for updates.
+      # If env var is specified and valid, override.
+      timeout = 600
+      env_var = ENV['CD4PE_MODULE_DEPLOY_READ_TIMEOUT']
+      unless env_var.nil?
+        if env_var.is_a?(Integer)
```

Comment:
can `#empty?` be truthy here?

Reply:
based on our zoom convo, this is not an issue